Gopkg.toml

```diff
@@ -44,7 +44,7 @@ required = [
 # and https://github.com/percona/percona-toolkit/pull/404.
```

Comment:
We should update that comment: PRs are merged, but there is no released with them yet.